src/main/java/org/jabref/logic/shared/event/SharedEntriesNotPresentEvent.java

```diff
@@ -0,0 +1,25 @@
+package org.jabref.logic.shared.event;
+
+import java.util.List;
+
+import org.jabref.model.entry.BibEntry;
+
+/**
+ * A new {@link SharedEntriwaNotPresentEvent} is fired, when the user tries to push changes of one or more obsolete
```

Comment:
Please fix typo `SharedEntriesNotPresentEvent`. Maybe, replace the text by "This event is fired ..."

Reply:
Makes sense. No need to reference the containing class.

Reply:
Done.